ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,91 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
+import { openSuccess } from '../alerts'
 
 const actions = {
   INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_PENDING: 'inventory/pending',
+  INVENTORY_CREATE: 'inventory/create',
+  INVENTORY_UPDATE: 'inventory/update',
+  INVENTORY_DELETE: 'inventory/delete',
+  INVENTORY_REFRESH: 'inventory/refresh'
 }
 
 export let defaultState = {
-  all: []
+  all: [],
+  fetched: false,
 }
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
-})
+export const findInventory = createAction(actions.INVENTORY_GET_ALL, () =>
+  (dispatch, getState, config) => axios
+    .get(`${config.restAPIURL}/inventory`)
```

Comment:
`restApiURL` is not the same as `restApiUrl`. Replace any instance of `restApiURL` with `restApiUrl`.

This can be viewed through the browser's developer tools by seeing an Error 404 in the Console.
This can also be seen when running the front end and looking at the Inventory page (after loading the sample data; refer to the project's README.adoc); you should expect to see inventory objects on the page.